zeq/subscriber.cpp

```diff
@@ -51,7 +49,7 @@ class Subscriber
 
     ~Subscriber()
     {
-        BOOST_FOREACH( SocketType socket, _subscribers )
+        for( const SocketType& socket : _subscribers )
```

Comment:
If this is the key pair type used in a map, I think it's preferable to use MapType::mapped_type (not sure about the spelling) or even auto (I think auto works here)